### ISSUE TITLE
Framework: Autotester stuck on PR #11547

### ISSUE THREAD

**Issue by GrahamBenHarper** [trilinos/Trilinos#11552]
## Bug Report
@trilinos/framework @sebrowne 
I pushed a commit right when the AT started testing #11547, and I think it broke. seeing as it has been testing for 19 hours, but #11549 has not yet started testing.

My PR isn't critical since it's just documentation, so it's fine if the testing doesn't go through and the AT gets rebooted or whatever is needed. I just want to point it out before the PRs in the lane pile up.

**Comment by sebrowne** [comment]
Excellent call, thank you!  The Java version requirement was updated yesterday on the automation server, so I had to manually configure all of the nodes to use a newer Java.  They're all back online now and running jobs, I'll report back on this tomorrow assuming that stuff runs as expected.  I wouldn't be surprised if it has lost track of stuff and anything has been partially running doesn't work the first time.
